src/assets/scripts/client/parsers/scopeCommandParser/ScopeCommandModel.js

```diff
@@ -0,0 +1,66 @@
+import _first from 'lodash/first';
+import _has from 'lodash/has';
+import _last from 'lodash/last';
+import _without from 'lodash/without';
+import _values from 'lodash/values';
+import {
+    IMPLIED_COMMANDS,
+    EXPLICIT_COMMANDS,
+    COMMAND_FUNCTIONS
+} from './scopeCommandMap';
+
+// FIXME: Remove dummy sector codes
+const SECTOR_HANDOFF_CODES = ['18', '19', '10', '12'];
+
+/**
+ * A command to be interpreted by the scope itself
+ * Used for manipulation of the display and other ATC tasks
+ *
+ * @class ScopeCommand
+ */
+export default class ScopeCommand {
+    constructor(command) {
+        this.aircraftReference = '';
+        this.commandArguments = [];
+        this.commandFunction = '';
+
+        this._init(command);
+    }
+
+    _init(command) {
+        const pieces = command.toUpperCase().split(' ');
+
+        this.aircraftReference = _last(pieces);
+        this.commandFunction = this._extractCommandFunction(pieces);
+        this.commandArguments = this._extractCommandArguments(pieces);
+    }
+
+    _extractCommandArguments(command) {
+        if (_values(IMPLIED_COMMANDS).indexOf(this.commandFunction) !== -1) {
```

Comment:
I've not used `_values()` before, is that like `Object.keys()` only for collecting the values of an object?

Reply:
Right. Can't use `_has()` because that gets the keys, and we need the values.